### ISSUE TITLE
WAN tested dedicated server ping shows up as grey/red ? in server browser

### ISSUE THREAD

**Issue by T-oasterO-ven** [Regalis11/Barotrauma#3305]
- [🗸] I have searched the issue tracker to check if the issue has already been reported.

**Description/Additional info**

**Intro:**
Recently, I created a dedicated server since steam P2P can get very laggy for other players on my server, especially since I live in a rural area without fiber internet, and with only ADSL2+ being the best around----other players frequently tend to lag out the entire server (300-500+ ping for everyone connected going as high as 1000+ ping----P2P is not good when compared to dedicated servers).

**The Issue:**
My "WAN tester" confirmed the grey question mark that showed up instead of a ping. However, when he joined, pressing TAB revealed the pings worked fine----the WAN tester also confirmed TAB pings worked perfectly. 

**Why this is a problem:**
After waiting for many hours for more than two people to connect, only one person connected then disconnected after 10 seconds. I've hosted identical P2P games that went full pretty quick, and none took this long. The wait is far beyond the average wait time for a P2P game when you start a server without inviting anyone. The grey question mark ping might be putting off "potential joiners"---maybe. But, after relaunching the game and checking the server browser almost all question mark servers are at the bottom of the list with 1-0 players.  

**Why this is happening (possibly):**
Many rural ISPs of "old" ADSL2+ tech block all ICMPv4 echo requests (standard ping), and some even block UDPv4 and TCPv4 pings for security reasons. Most people who live in rural areas will have this problem since there are still many rural areas without fiber internet, which leaves many stuck with DSL/ADSL as the best they can get (might be why I see a lot of ? mark servers). Some modems allow you to set options that allow pings, but many do not and just block pings by default. I was lucky to have the setting called "stealth mode" which ignores ICMP pings, unticking the box did not help.

**Attempts at a fix (WAN tester confirmed none have worked except last 'bullet'):**
-Unblocked ICMPv4 inbound/outbound requests in windows firewall.
-Checked router for any settings that would allow pings.
-Forwarded ports that might have helped (no luck--they are closed now).
-Disabled router "stealth mode" (which ignores pings of any type).
-Attempted to use the tools from https://www.netscantools.com/nstpro_traceroute.html
to see if any other ping types would work, none have (maybe that's only because I'm part of the LAN making up the WAN address I am pinging).

**What I can not attempt:**
-Flashing custom modem firmware (brick risk)
-Calling my ISP, since public IPs are given out to multiple clients and you have to pay a lot more for your own IP. They will not allow pings on shared WAN IPs.

**My suggestion**:
See if there is a way around this that is not blocked by most ISPs/modems. Since the TAB pings worked perfectly when the "WAN tester" client connected to the server. There may be a way to get a player's TAB ping to show up as the browser ping, so we don't have all these question marks as pings, which I have never had as a problem in my years hosting dedicated servers for other games. 

**Addendum:**
Thanks for reading, I hope this gets fixed soon in a small patch or as a part of the next big update.

Also, you probably are aware of this "sub getting stuck in outpost" bug for some subs (especially workshop subs like this one: https://steamcommunity.com/sharedfiles/filedetails/?id=2101353155)

If anyone requires more information or has any questions on anything above please let me know.
Thank you.

**TL; DR:** Tab pings work perfectly, but browser one does not, which leads to painfully slow join rate compared to P2P hosting. Fix is requested as my attempts at a fix have failed and P2P hosting in my rural area is terrible compared to dedicated servers.

**Steps To Reproduce**
Properly set up a dedicated server following the pinned steam guide, then get a friend outside of the LAN to test server browser ping and TAB ping.

ISP: CenturyLink 
Internet: ADSL2+
Modem: Zyxel PK5001Z

**Version**
0.9.10.0 Windows 10

"Captain we've _pinged_ some bugs in the water!"

**Comment by Rokvach** [comment]
This has already been fixed in the next patch, closing.
